addons/sourcemod/scripting/AntiBhopCheat.sp

```diff
@@ -431,7 +490,13 @@ void NotifyAdmins(int client, const char[] sReason)
 		}
 	}
 
-	Forward_OnDetected(client, sReason, g_sStats);
```

Comment:
why did you remove the function

Reply:
Making a function of something who is used only time is not usefull.